logverification/app/massif.go

```diff
@@ -0,0 +1,41 @@
+package app
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/datatrails/go-datatrails-merklelog/massifs"
+)
+
+const (
+	contextTimeout = 30 * time.Second
+)
+
+var (
+	ErrNilMassifContext = errors.New("nil massif context")
+)
+
+type MassifGetter interface {
+	GetMassif(
```

Comment:
MassifGetter.GetMassif is very verbose.

Why not just Massif.Get?

Reply:
It looks verbose where we're defining the interface, but its possible that clients of this interface will want to call the variable something other than MassifGetter, so the verbosity could still be helpful

Reply:
it is defined by the massifs module in go-datatrails-merklelog, as well as a forward declared interface in veracity (because we do the offline log stuff in veracity) so changing the name from `GetMassif` to `Massif` is more involved than i think is needed or this ticket?